apps/vault/contracts/Vault.sol

```diff
@@ -112,6 +112,6 @@ contract Vault is EtherTokenConstant, AragonApp, DepositableStorage {
     *      maliciously to transfer funds away from the vault
     */
     function allowRecoverability(address) public view returns (bool) {
-        return false;
+        return hasInitialized() ? false : true;
```

Comment:
I was thinking about this, and wondering what would be best for native ETH. If the vault's not initialized, it should be impossible to deposit ETH into the contract anyway, so it might make sense to say that we disallow recovering ETH too.

Otherwise, we could just do a `!hasInitialized()` here.